From: Stefano Rivera <stefanor@debian.org>
Date: Fri, 16 Jul 2021 20:28:27 -0400
Subject: Drop dependency on backports.entry-points-selectable

Not packaged in Debian, instead we can just pin a dependency on
importlib_metadata >= 3.6

Forwarded: not-needed
---
 setup.cfg                            | 1 -
 src/virtualenv.egg-info/requires.txt | 1 -
 src/virtualenv/run/plugin/base.py    | 2 +-
 3 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/setup.cfg b/setup.cfg
index 70d77fb..1cec841 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -40,7 +40,6 @@ project_urls =
 [options]
 packages = find:
 install_requires = 
-	backports.entry_points_selectable>=1.0.4
 	distlib>=0.3.1,<1
 	filelock>=3.2,<4
 	platformdirs>=2,<3
diff --git a/src/virtualenv.egg-info/requires.txt b/src/virtualenv.egg-info/requires.txt
index a1c509f..4a59593 100644
--- a/src/virtualenv.egg-info/requires.txt
+++ b/src/virtualenv.egg-info/requires.txt
@@ -1,4 +1,3 @@
-backports.entry_points_selectable>=1.0.4
 distlib<1,>=0.3.1
 filelock<4,>=3.2
 platformdirs<3,>=2
diff --git a/src/virtualenv/run/plugin/base.py b/src/virtualenv/run/plugin/base.py
index f1f4ee0..65c0f2d 100644
--- a/src/virtualenv/run/plugin/base.py
+++ b/src/virtualenv/run/plugin/base.py
@@ -2,7 +2,7 @@ from __future__ import absolute_import, unicode_literals
 
 from collections import OrderedDict
 
-from backports.entry_points_selectable import entry_points
+from importlib_metadata import entry_points
 
 
 class PluginLoader(object):
